Cargo.toml

```diff
@@ -52,12 +52,14 @@ serde = { version = "1.0", default-features = false, optional = true, features =
 packed_simd = { version = "0.3.4", package = "packed_simd_2", features = ["into_bits"], optional = true }
 zeroize = { version = "1", default-features = false }
 fiat-crypto = { version = "0.1.6", optional = true}
+group = { version = "0.11", default-features = false, optional = true }
```

Comment:
It would be nice to support some of the rest of the types if possible since that would allow using it in the rest of the tooling the `elliptic_curve` crate has, like their [`PublicKey`](https://docs.rs/elliptic-curve/latest/elliptic_curve/struct.PublicKey.html) serialization/deserialization logic.